qa/workunits/windows/test_rbd_wnbd.py

```diff
@@ -616,6 +665,30 @@ def print_results(cls,
             print(table)
 
 
+class RbdResizeFioTest(RbdFioTest):
+    """Image resize test.
+
+    This test extends and then shrinks the image, performing FIO tests to
+    validate the resized image.
+    """
+
+    @Tracer.trace
+    def run(self):
+        self.image.resize(self.image_size_mb * 2)
+        self.image.wait_for_disk_resize()
+
+        self._run_fio(fio_size_mb=self.fio_size_mb * 2)
```

Comment:
I think this should be  `self.image_size_mb * 2` instead of `self.fio_size_mb * 2`.  The string of the "passed value or fallback" sizes is getting a bit convoluted.  Where the non-defaulted `fio_size_mb` can come from initially, for example?

We want to generate I/O against the whole image here so let's be explicit about that (even in spite of the fact that without out-of-bounds errors we aren't going to get any real feedback).

Reply:
Indeed, I went with ``self.fio_size_mb`` for consistency but it makes more sense to use ``self.image_size_mb`` explicitly.

Reply:
> The string of the "passed value or fallback" sizes is getting a bit convoluted. Where the non-defaulted fio_size_mb can come from initially, for example?

It defaults to ``self.image_size_mb``: https://github.com/ceph/ceph/blob/6d583d25c5c0d6cef470e81d00a590ed03f1cd8e/qa/workunits/windows/test_rbd_wnbd.py#L509-L520

The Fs FIO test takes the FS size overhead into account: https://github.com/ceph/ceph/blob/6d583d25c5c0d6cef470e81d00a590ed03f1cd8e/qa/workunits/windows/test_rbd_wnbd.py#L623-L626